docs/source/tools_integration/ipython.md

```diff
@@ -1,173 +1,168 @@
 
-## Kedro and Jupyter
+The Kedro IPython extension makes the following variables available in your IPython or Jupyter session:
 
-You may want to use Jupyter notebooks to experiment with your code as you develop new nodes for a pipeline, although you can write them as regular Python functions without a notebook. To use Kedro's Jupyter session:
+* `catalog` (type `DataCatalog`): [Data Catalog](../data/data_catalog.md) instance that contains all defined datasets; this is a shortcut for `context.catalog`
+* `context` (type `KedroContext`): Kedro project context that provides access to Kedro's library components
+* `pipelines` (type `Dict[str, Pipeline]`): Pipelines defined in your [pipeline registry](../nodes_and_pipelines/run_a_pipeline.md#run-a-pipeline-by-name)
+* `session` (type `KedroSession`): [Kedro session](../kedro_project_setup/session.md) that orchestrates a pipeline run
 
-```bash
-kedro jupyter notebook
-```
+We will now give some examples of how these variables can be used in the [Iris example project](../get_started/example_project.md). To explore the full range of attributes and methods available, you might like to consult the relevant [API documentation](/kedro) or use [the Python `dir` function](https://docs.python.org/3/library/functions.html#dir) (e.g. `dir(catalog)`).
 
-This starts a Jupyter server and opens a window in your default browser.
+### `catalog`
 
-```eval_rst
-.. note::  If you want Jupyter to listen to a different port number, then run ``kedro jupyter notebook --port <port>``.
-```
+`catalog` can be used to explore your [Data Catalog](../data/data_catalog.md), including parameters. Useful methods include `catalog.list`, `catalog.load` and `catalog.save`. For example:
 
-Navigate to the `notebooks` folder of your Kedro project and create a new notebook.
+```ipython
+In [1]: catalog.list()
+Out[1]:
+['example_iris_data',
+ 'parameters',
+ 'params:example_test_data_ratio',
+ 'params:example_num_train_iter',
+ 'params:example_learning_rate']
+
+In [2]: catalog.load("example_iris_data")
+Out[2]:
+     sepal_length  sepal_width  petal_length  petal_width    species
+0             5.1          3.5           1.4          0.2     setosa
+1             4.9          3.0           1.4          0.2     setosa
+2             4.7          3.2           1.3          0.2     setosa
+3             4.6          3.1           1.5          0.2     setosa
+4             5.0          3.6           1.4          0.2     setosa
+..            ...          ...           ...          ...        ...
+145           6.7          3.0           5.2          2.3  virginica
+146           6.3          2.5           5.0          1.9  virginica
+147           6.5          3.0           5.2          2.0  virginica
+148           6.2          3.4           5.4          2.3  virginica
+149           5.9          3.0           5.1          1.8  virginica
+
+In [3]: catalog.load("parameters")
```

Comment:
Is this the only way that users can access parameters now? 🥴 This looks confusing that you would be using `catalog.load` to find parameters.

Reply:
I think you can also do `context.params`